### PR TITLE
Added section about max-input-vars

### LINKED ISSUE #1527: Add the recommended value for php.ini variable `max_input_vars`
Feedback on page: /guides/v2.1/install-gde/prereq/php-settings.html

Please see https://github.com/magento/magento2/issues/9317#issuecomment-341350686

Apparently, according to what seems to be an official response from the Community Engineering Team, it's a "correct Magento behavior" to have over 1000 html inputs on the product edit page in admin. Unfortunately, this behavior leads to an obscure error when using default PHP settings.

Please provide the recommended value for this `max_input_vars` variable in the docs (wherever appropriate and possible) so that this setting could be preset during Magento installation, in order to avoid the error mentioned above.

Thanks in advance & have a nice day!

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:

- [ ] New topic
- [x ] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will...

Add a section about max-input-vars to the php troubleshooting page..

Fixes #1527 

whatsnew
Added a section to the PHP troubleshooting guide discussing max-input-vars.

### CODE REVIEW

File: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md [hunk 1]
```diff
@@ -85,4 +98,6 @@ Multiple templates have syntax for support abstract level on templates (use diff
 
 More information about [asp_tags](http://php.net/manual/en/ini.core.php#ini.asp-tags){:target="_blank"}.
 
-Edit `php.ini` and set `asp_tags = off`. For more information, see [Required PHP settings]({{ page.baseurl }}/install-gde/prereq/php-settings.html).
+Edit `php.ini` and set `asp_tags = off`. For more information, see [Required PHP settings][]
```
Comment (shrielenee) [inline guides/v2.1/install-gde/trouble/php/tshoot_php-set.md]: Add a period at the end of this sentence.

File: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md [hunk 2]
```diff
@@ -62,6 +62,19 @@ To increase your PHP memory limit:
 	*	nginx (both CentOS and Ubuntu): `service nginx restart`
 8.	Try the installation again.
 
+
+### max-input-vars error due to large forms
+
+Some configurations with a high number of storeviews, products, attributes, or options may generate forms that exceed a PHP limit.
+If the number of values being sent surpasses the `max-input-vars` limit set within `php.ini` (default is 1000), some data will not be transferred. This limit may manifest by some values not being updated in the database. 
+There may also be a warning in the PHP log:
```
Comment (jcalcaben) [inline guides/v2.1/install-gde/trouble/php/tshoot_php-set.md]:
```suggestion
When this occurs, a warning appears in the PHP log:
```
Make this sentence less ambiguous.

File: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md [hunk 3]
```diff
@@ -62,6 +62,19 @@ To increase your PHP memory limit:
 	*	nginx (both CentOS and Ubuntu): `service nginx restart`
 8.	Try the installation again.
 
+
+### max-input-vars error due to large forms
+
+Some configurations with a high number of storeviews, products, attributes, or options may generate forms that exceed a PHP limit.
```
Comment (jcalcaben) [inline guides/v2.1/install-gde/trouble/php/tshoot_php-set.md]:
```suggestion
Configurations with a high number of storeviews, products, attributes, or options can generate forms that exceed the preset PHP limit.
```

File: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md [hunk 4]
```diff
@@ -62,6 +62,19 @@ To increase your PHP memory limit:
...
+If the number of values being sent surpasses the `max-input-vars` limit set within `php.ini` (default is 1000), some data will not be transferred. This limit may manifest by some values not being updated in the database.
```
Comment (jcalcaben) [inline guides/v2.1/install-gde/trouble/php/tshoot_php-set.md]:
```suggestion
If the number of values sent surpasses the `max-input-vars` limit set within `php.ini` (default is 1000), the remaining data is not transferred and those database values do not get updated.
```

File: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.1/install-gde/trouble/php/tshoot_php-set.md]: Done.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
Something went astray and 4 more files were mysteriously added to the PR. Closing and redoing.

**Pull request closed without merging**